contrib/freshmaker-cli

```diff
@@ -225,6 +244,42 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns
+    the list of advisories that are IMPORTANT or CRITICAL.
+    """
+    krb_auth = HTTPKerberosAuth()
+    r = requests.get(errata_url, verify=False, auth=krb_auth)
+    r.raise_for_status()
+    data = r.json()
+    return [
+        advisory["id"]
+        for advisory in data
+        if advisory["security_impact"] in ("Important", "Critical")
+    ]
+
+
+def get_freshmaker_events_by_search_keys(api_url, search_keys):
+    print("Getting Freshmaker events")
+    events = []
+    for search_key in search_keys:
+        url = urllib.parse.urljoin(api_url, f'/api/1/events/?search_key={search_key}')
+        r = requests.get(url, timeout=30)
+        r.raise_for_status()
+        ret = r.json()
+        if ret["items"]:
+           events += ret["items"]
```

Comment:
IMHO it's maybe clearer to get just the ones that were missed, that was the point of the card. This should substitute the manual process of going to ET, check the latest shipped advisories that are important and critical and then for each one of them check Freshmaker's API.... The outcome should be just the ones that were ignored in the end, I don't care that much about the ones that were processed, otherwise I'd still have to "manually" filter out those ones.

Reply:
@gnaponie . PTAL now